SurfaceTopography/Uniform/Derivative.py

```diff
@@ -366,45 +365,49 @@ def fourier_derivative(topography, imtol=None):
 
     Parameters
     ----------
-    topography : :obj:`SurfaceTopography` or :obj:`UniformLineScan`
+    self : :obj:`SurfaceTopography` or :obj:`UniformLineScan`
         Surface topography object containing height information.
-    imtol: float, optional
-        tolerance for the discarded imaginary part. If the maximum absolute of
-        the imaginary part of the interpolated topography is more then that
-        value times the total absolute value of dx, an AssertionError is raised
-        If not specified the assertion will not be made.
+    n : int
```

Comment:
please either implement or remove of the docstring

Reply:
Okay, I removed the docstring